app/controllers/stories_controller.rb

```diff
@@ -85,12 +87,25 @@ def import
   end
 
   def export
-    csv = CSV.generate(headers: true) { |csv|
-      csv << CSV_HEADERS
-      @project.stories.by_position.each do |story|
-        csv << story.attributes.slice(*CSV_HEADERS)
+    csv = if params[:export_with_comments] == "1"
+      CSV.generate(headers: true) do |csv|
+        csv << CSV_HEADERS.append("comment")
```

Comment:
this is creating a problem, we should never modify constants

every time an export is done with comments, a new `comment` column is added to the constant, so we keep adding columns until the server is restarted

![image](https://github.com/fastruby/points/assets/188464/f67d0cec-d134-4395-8eff-f4fef8cfe18c)

Then, because the constant is changed, exporting without comments also creates `comment` columns